appwrite/appwrite
# Add events for functions and executions

## What does this PR do?

This PR adds events for the following actions in the API:
- `functions.create`
- `functions.update`
- `functions.delete`
- `functions.tags.create`
- `functions.tags.update`
- `functions.tags.delete`
- `functions.executions.create`

It also adds the following event in the Functions worker:
- `functions.executions.update`

## Test Plan

- Create webhook for new events
- Trigger the events through the console
- Observe logs for `appwrite-worker-webhooks` for webhook triggers
- Observe webhook target endpoint (`request-catcher` in testing)

## Todo

- [X] Create events
- [x] Create webhook tests

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

yes.
